fractal_server/app/api/v1/project.py

```diff
@@ -320,6 +320,10 @@ async def apply_workflow(
         input_dataset_id=input_dataset_id,
         output_dataset_id=output_dataset_id,
         workflow_id=workflow_id,
+        workflow_dump=dict(
+            workflow.dict(),
```

Comment:
Can we exclude `task_list` explicitly, as in `workflow.dict(exclude={"task_list"})`? It'd be to make it a bit more transparent.

Reply:
ok